mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -758,37 +761,42 @@ func testAccMongoDBAtlasClusterAWSConfigdWithLabels(projectID, name, backupEnabl
 	`, projectID, name, backupEnabled, tier, region, labelsConf)
 }
 
-func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
+func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
```

Comment:
could change the name of the function to have more coherency with the other ones?

Reply:
done